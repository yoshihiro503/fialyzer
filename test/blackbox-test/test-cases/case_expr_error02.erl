%% Dialyzer does not judge this code as an error.
-module(case_expr_error02).

-export([main/1]).

main(X) ->
    case X of
        1 -> 'foo';
        ok -> 123
    end + 42.
