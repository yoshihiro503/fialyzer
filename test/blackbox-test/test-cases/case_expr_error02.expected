TODO:filename:-1: Type error: type mismatch;
  found   : number()
  required: 'foo'
there is no solution that satisfies subtype constraints
